lib/ljglibs/gobject/object.moon

```diff
@@ -7,9 +7,9 @@ C, ffi_cast, ffi_new = ffi.C, ffi.cast, ffi.new
 lua_value = types.lua_value
 
 core.define 'GObject', {
-  new: (type) ->
+  new: (type, ...) ->
```

Comment:
Good catch! There's an additional twist when invoking functions though - the arg list needs to be `nil`-terminated or bad things will happen (see e1bec1c).